test/cobol/koopa/cobol/grammar/preprocessing/test/ReplaceStatement.stage

```diff
@@ -6,6 +6,8 @@ target replaceStatement;
 +[ REPLACE               ==foo bar== BY ==baz== . ]
 +[ REPLACE               ==foo bar== BY ====    . ]
 
++[ REPLACE               ==:foo:== BY ==bar==    . ]
```

Comment:
This just tests the parser which, happily, already accepts the example.

I'll try to push you a preprocessor test which shows the actual problem you're trying to fix.